tests/events.rs

```diff
@@ -0,0 +1,35 @@
+use std::time::Duration;
+
+use mio::{Events, Poll, Token, Waker};
+
+const WAKE_TOKEN: Token = Token(10);
+
```

Comment:
Can you move the `event_source.rs` tests into this file? We're have too many tests file as it is and the file name is better.

Reply:
ok, done.